arboretum/common/iam_ibm.py

```diff
@@ -0,0 +1,22 @@
+"""Utility function for IBM Cloud IAM."""
```

Comment:
You're missing the copyright notice and this should be:

```python
"""Utility module for IBM Cloud IAM."""
```

Reply:
thank you, I added copyright notice and update the docstring as you proposed.

Reply:
thank you for pointing out, I added the copyright notice, and updated the docstring as you proposed.